osaca/osaca.py

```diff
@@ -833,37 +819,35 @@ def main():
                        help='search for IACA markers instead the OSACA marker')
     group.add_argument('-m', '--insert-marker', dest='insert_marker', action='store_true',
                        help='try to find blocks probably corresponding to loops in assembly and'
-                       + 'insert IACA marker')
+                            + 'insert IACA marker')
     parser.add_argument('filepath', type=str, help='path to object (Binary, ASM, CSV)')
 
     # Store args in global variables
     inp = parser.parse_args()
-    if(inp.arch is None and inp.insert_marker is None):
```

Comment:
In case of starting OSACA with `-m`  flag (for inserting IACA markers with kerncraft), no architecture is necessary. Therefore the check is valid. Added with the next commit.